bin/hook-docs/data/actions.json

```diff
@@ -61,6 +61,54 @@
                 "long_description_html": "<p>This hook fires when an item is added to the cart. This is triggered from the Store API in this context, but WooCommerce core add to cart events trigger the same hook.</p>"
             }
         },
+        {
+            "name": "woocommerce_after_main_content",
+            "file": "BlockTypes/LegacyTemplate.php",
+            "type": "action",
+            "doc": {
+                "description": "Hook: woocommerce_after_main_content.",
+                "long_description": "",
+                "tags": [
+                    {
+                        "name": "hooked",
+                        "content": "woocommerce_output_content_wrapper_end - 10 (outputs closing divs for the content)"
+                    }
+                ],
+                "long_description_html": ""
+            }
+        },
+        {
+            "name": "woocommerce_after_main_content",
```

Comment:
This is a duplicate of the above. It's also unrelated to the hooks you added in this PR. Is this something that exists but was missing from the docs?